<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Composition - Resource Hub</title>
  <link rel="stylesheet" href="../../style.css">
</head>
<body>
  <header>
    <div class="logo"><a href="../../index.html">Resource Hub</a></div>
    <nav>
      <ul>
        <li><a href="../../index.html">Home</a></li>
        <li><a href="../../index.html#categories">Categories</a></li>
        <li><a href="../../aboutUs.html">About Us</a></li>
      </ul>
    </nav>
    <div class="search-container">
      <input type="text" id="search-bar" placeholder="Search categories..." onkeyup="searchSite()">
      <div id="search-results" class="search-results"></div>
    </div>
  </header>

  <div class="breadcrumbs">
    <a href="../../index.html">Home</a> ›
    <a href="../../index.html#categories">Categories</a> ›
    <a href="dnaSequenceAnalysis.html">DNA Sequence Analysis</a> › 
    <span>Composition</span>
    
  </div>

  <main class="content">
    <h1>Composition</h1>

    <div class="software-full-width-image">
    <img src="../DNAsequenceAnalysis/pg3picsDNA/compositionTable.png" alt="SnapGene software">
    </div>
    <br>    

    <p>A good internet resource on GC content can be found on the <a href = "https://www.bionity.com/en/encyclopedia/GC-content.html">bionity.com</a> website.

</p>

    <ul>
      <li><a href = "https://www.ncbi.nlm.nih.gov/tools/vecscreen/">VecScreen</a> (National Center for Biotechnology Information) - screens your DNA sequence for potential vector sequence.  Well worth running before doing any other analysis.</li>
        <br>
        <li>Base composition - consider <a href = "https://www.bioinformatics.nl/cgi-bin/emboss/wordcount#forms::wordcount">WORDCOUNT</a>  (EMBOSS Suite) which gives one the option of choosing the "word size".  The latter provides a nice output of mono-, di- and trinucleotide frequencies. Select "create statistics" and "start task" to get to the sequence entry page.</li>
        <br>
        <li><a href = "http://gscompare.ehu.eus/tools/oligo_frequencies/">Oligonucleotide frequencies calculator</a> - in addition to calculating the number and frequency of monomeric to octomeric sequences this site also will calculate Zero Order Markov chain for tetranucleotides (ZOM);  First Order Markov chain for tetranucleotides (FOM); Second Order Markov chain for tetranucleotides (SOM); and, Z-scores for tetranucleotides.</li>
        <br>
        <li><a href = "http://gscompare.ehu.eus/tools/compareseqs/">Compare oligonucleotide frequencies</a> among sequences - This tool will compute distance between input sequences and UPGMA clustering will be display in tabular and dendogram formats.</li>
        <br>
        <li><a href = "https://dnasv2.kamalrawal.in/">DNASCANNER v2</a> - offers an extensive analysis of 158 DNA properties, including mono/di/trinucleotide frequencies, structural, physicochemical, thermodynamics, and mechanical properties of DNA sequences. The tool provides downloadable results and offers interactive plots for easy interpretation and comparison between different features. 
          <br>(<strong>Reference:</strong> Preeti P et al. 2024. J Comput Biol 31(7): 651-669).</li>
        <br>
        <li><a href = "https://www.sciencebuddies.org/science-fair-projects/references/genomics-g-c-content-calculator">Genomics %G~C Content Calculator</a> (Science Buddies.org) - simple calculator for mol%G+C plus counts the individual bases. </li>
        <br>
        <li>Compositional heterogeneity: <a href = "https://www.endmemo.com/bio/gcdraw.php">GC Content Plot Online.</a>  Similar site <a href = "https://jamiemcgowan.ie/bioinf/gc_content_plot.html">here.</a></li>
        <br>
        <li><a href = "https://4virology.net/virology-ca-tools/">GraphDNA</a> - DNA Skew Graphing (Viral Bioinformatics Resource Center, University of Victoria, Canada) - this Java applet performs DNA walks, purine, AT and GC skews on small (&lt1 Mb) genomes. Requires registration and login. Alternative locations for cumulative GC skew are the <a href = "https://gcat.davidson.edu/DGPB/gc_skew/gc_skew.html">GC  Skewing</a> (Davidson College, U.S.A.).  Genomic nucleotide skews  are explained <a href = "https://nycdatascience.com/blog/student-works/using-the-gc-skew-to-analyze-bacterial-genome-evolution/">here.</a></li>
        <br>
        <li><a href = "https://www.biologicscorp.com/tools/GCContent/#.WM116aLatPY">GC Content Calculator</a>  (Biologics International Corp, Indianapolis, USA)  - DNA GC-content percentage is calculated as Count(G + C)/Count(A + T + G + C) * 100. This program was used to generate the following disgram of Escherichia phage lambda (NC_001416) using a window of 48 bp. One can click on the peaks and valleys and get a read-out of the localized GC-content.  Another graphically good site is on the <a href = "https://en.vectorbuilder.com/tool/gc-content-calculator.html">VectorBuilder</a> site.</li>
            <div class="software-full-width-image">
            <img src="../DNAsequenceAnalysis/pg3picsDNA/Biologicscorp.gif" alt="SnapGene software">
            </div>
            <br>  

        <li><a href = "https://www.bioinformatics.org/jambw/">JaMBW</a> (European Molecular Biology Laboratory of Heidelberg, Germany). Java based Molecular Biologist's Workbench.Select Chapter 1 for sequence format conversion (upper <-> lower case; T <-> U; reverse or complement sequence).  N.B. Also check out   Chapter 5 "Buffer Calculator."  </li>
        <br>
        <li><a href = "https://users-birc.au.dk/~palle/php/fabox/random_sequence_generator.php">Random DNA sequence generator</a> 
          <br>(<strong>Reference:</strong> Villesen, P. 2007.  Molecular Ecology Notes 7: 965–968.).  
          <br>Similar resources are available <a href = "https://faculty.ucr.edu/~mmaduro/random.htm">here</a> and <a href = "https://birc.au.dk/~palle/php/fabox/random_sequence_generator.php">here.</a></li>
        <br>
        <li><a href = "https://www.lri.fr/genrgens/">GenRGenS</a>, a software dedicated to random generation of genomics sequences that supports several classes of models, including Markov chains, HMM, context-freegrammars, PROSITE patterns and more. 
          <br>(<strong>Reference:</strong> Y. Ponty et al. Bioinformatics, 22:1534-1535).</li>
        <br>
    </ul>

    </section>

    <p class="updated"><strong>Updated:</strong> December, 2025</p>
  </main>

  <footer>
      <p class="credits">
        Designed and developed by <strong>Um-E-Kalsoum Asif</strong>
      </p>
      <p>© 2025 Internet Resources for Molecular Biologists</p>


    </footer>

</body>
</html>
